### PR TITLE
Fixed creating species with the same name

### PR DESCRIPTION
**Pull request by tranquanghuy0801**
Resolves issue #58 

- Add a notification "Species already exist" if user create new species with the same name
- Tagging species in a capture already created new species if they did not exist

### CODE REVIEW

File: src/components/SpeciesTable.js [hunk 1]
```diff
@@ -384,14 +391,19 @@ const EditModal = ({
...
   const handleSave = async () => {
-    setIsEdit(false);
-    await editSpecies({
-      id: speciesEdit.id,
-      name: speciesEdit.name,
-      desc: speciesEdit.desc,
-    });
-    loadSpeciesList();
-    setSpeciesEdit(undefined);
+    if (nameSpecies.includes(speciesEdit.name.trim())) {
```
Comment (nmcharlton) [inline src/components/SpeciesTable.js:394]: Can we make this a case-insensitive comparison?

**Review by nmcharlton — CHANGES_REQUESTED** [review]
This looks pretty good for the species table, just one suggestion about case-insensitive name matching.

We also need to make the `isNewSpecies` check in `src/models/species.js` case-insensitive.

File: src/components/SpeciesTable.js (hunk 1 above)
Comment (tranquanghuy0801) [reply src/components/SpeciesTable.js]: Yes. I have fixed in my latest commit.

**Review by nmcharlton — APPROVED** [review]
Awesome work 🌴

**Pull request merged**